packages/spec/README.md

```diff
@@ -1,22 +1,19 @@
 # Standard Schema Spec
```

Comment:
Most people looking at the README probably care most about the interface itself, so I reorganized it to move that toward the top.

Also changed the root-level README to a symlink to keep it in sync. I've used this in the past, but not 100% sure if this works well everywhere.

If not, perhaps we should just have a repo build step that copies one to the other or asserts that they're the same?

Reply:
> Most people looking at the README probably care most about the interface itself, so I reorganized it to move that toward the top.

I think the current structure tells the story behind the project better, but in the end it probably won't make much of a difference. I'm fine with either.

> Also changed the root-level README...

In general, I prefer to keep things simple, but since we are likely to update the ecosystem section of the README more frequently if  Standard Schema works out a symlink (if supported by GitHub, npm and JSR) sounds like a good solution.